platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/eventhandling/TransactionPoolMetrics.java

```diff
@@ -64,14 +66,14 @@ public TransactionPoolMetrics(
 
         metrics.getOrCreate(new FunctionGauge.Config<>(
                         PLATFORM_CATEGORY, "bufferedTransactions", Integer.class, getBufferedTransactionCount)
-                .withDescription("transEvent queue size")
+                .withDescription("The number of transactions waiting to be inserted into an event.")
```

Comment:
Is it worth specifying that these are "non-priority" transactions?

Reply:
updated wording